docs/Tiered-Storage-on-Tachyon.md

```diff
@@ -1,90 +1,208 @@
+freed. Tachyon supports custom evictors, and implementations include:
+
+* GREEDY: evicts arbitrary blocks until the required size is freed
+* LRU: evicts the least-recently-used blocks until the required size is freed
+
+In the future, additional evictors will be available. Since Tachyon supports custom evictors,
+you can also develop your own evictor appropriate for your workload.
 
 Currently only synchronous eviction is supported by tiered storage. It is recommended to use
 small block size (less than 64MB), to reduce the latency of block eviction. This restriction will
 not exist when asynchronous eviction is introduced.
+
+# Enabling and Configuring Tiered Storage
+
+Tiered storage can be enabled in Tachyon with some
+[configuration parameters](Configuration-Settings.html). By default, Tachyon only enables a single,
+MEM tier. To specify additional tiers for Tachyon, use the following configuration parameters:
+
+    tachyon.worker.tieredstore.level.max
+    tachyon.worker.tieredstore.level{x}.alias
+    tachyon.worker.tieredstore.level{x}.dirs.quota
+    tachyon.worker.tieredstore.level{x}.dirs.path
+
+For example, if you wanted to configure Tachyon to have two tiers, of MEM and HDD, it would be
+configured something like:
+
+    tachyon.worker.tieredstore.level.max=2
+    tachyon.worker.tieredstore.level0.alias=MEM
+    tachyon.worker.tieredstore.level0.dirs.path=/mnt/ramdisk
+    tachyon.worker.tieredstore.level0.dirs.quota=100GB
+    tachyon.worker.tieredstore.level1.alias=HDD
+    tachyon.worker.tieredstore.level1.dirs.path=/mnt/hdd1,/mnt/hdd2,/mnt/hdd3
+    tachyon.worker.tieredstore.level1.dirs.quota=2TB,5TB,500GB
+
+Here is the explanation of the example configuration:
+
+* `tachyon.worker.tieredstore.level.max=2` configures 2 tiers in Tachyon.
+* `tachyon.worker.tieredstore.level0.alias=MEM` configures the first (top) tier to be the MEM tier.
+* `tachyon.worker.tieredstore.level0.dirs.path=/mnt/ramdisk` defines `/mnt/ramdisk` to be the file
+path to the ramdisk.
+* `tachyon.worker.tieredstore.level0.dirs.quota=100GB` sets the quota for the ramdisk to be `100GB`.
+* `tachyon.worker.tieredstore.level1.alias=HDD` configures the second tier to be HDD.
+* `tachyon.worker.tieredstore.level1.dirs.path=/mnt/hdd1,/mnt/hdd2,/mnt/hdd3` configures 3 separate
+file paths for the HDD tier.
+* `tachyon.worker.tieredstore.level1.dirs.quota=2TB,5TB,500GB` defines the quota for each of the 3
+file paths of the HDD tier.
+
+There are a few restrictions to defining the tiers. First of all, there can be at most 3 tiers.
+Also, at most 1 tier can refer to a specific alias. For example, at most 1 tier can have the alias
+HDD. If you want Tachyon to use multiple hard drives for the HDD tier, you can configure that as
+multiple storage directories in the HDD tier.
+
+Additionally, the specific evictor and allocator strategies can be configured. Those configuration
+parameters are:
+
+    tachyon.worker.allocate.strategy=MAX_FREE
```

Comment:
I think master branch now changed these (this one and the one below) to be "tachyon.worker.allocate.*.class", and with the class name.